Add TabWidget (#395)
sixtyfpsui/sixtyfps: docs/widgets.md

}
```

## `TabWidget`

> The container for a set of tabs. TabWidget can only have `Tab` elements as children. Only one tab will be rendered at
> the time.

Reviewer: ```suggestion
TabWidget is a container for a set of tabs. It can only have `Tab` elements as children and only one tab will be visible at
a time.
```
